src/editor.js

```diff
@@ -1806,4 +1806,19 @@ function testProjectEquality(projectA, projectB) {
     // yours: true
 
     return true;
-}
\ No newline at end of file
+}
```

Comment:
This is just a stub function to satisfy the callback requirement for the onClick event handler that references this function. We're doing this instead of ripping out code because we will likely need this when the custom code blocks are back in the project.